frontend/src/Pages/RecruitmentApplicationsOverviewPage/components/ActiveApplications/ActiveApplications.tsx

```diff
@@ -0,0 +1,142 @@
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { toast } from 'react-toastify';
+import { Button, Table } from '~/Components';
+import { getRecruitmentApplicationsForApplicant, withdrawRecruitmentApplicationApplicant } from '~/api';
+import type { RecruitmentApplicationDto } from '~/dto';
+import { KEY } from '~/i18n/constants';
+import { niceDateTime } from '~/utils';
+import type { ApplicantApplicationManagementQK } from '../../RecruitmentApplicationsOverviewPage';
+import { ActiveApplicationLink } from './ActiveApplicationLink';
+import { ControlPriorityButtons, type PriorityChange } from './ControlPriorityButtons';
+
+type ActiveApplicationsProps = {
+  recruitmentId?: string;
+  queryKey: ApplicantApplicationManagementQK;
+};
+
+export function ActiveApplications({ recruitmentId, queryKey }: ActiveApplicationsProps) {
+  const { t } = useTranslation();
+  const queryClient = useQueryClient();
+  const [recentChanges, setRecentChanges] = useState<PriorityChange[]>([]);
+
+  // Clear the recent change after 2 seconds
+  useEffect(() => {
+    if (recentChanges.length > 0) {
+      const timer = setTimeout(() => {
+        setRecentChanges([]);
+      }, 2000);
+      return () => clearTimeout(timer);
+    }
+  }, [recentChanges]);
+
+  // Query for fetching applications
+  const { data: applications = [] } = useQuery({
+    queryKey: ['applications', recruitmentId],
+    queryFn: () => getRecruitmentApplicationsForApplicant(recruitmentId as string).then((response) => response.data),
```

Comment:
Endre på getRecruitmentApplicationsForApplicant til å returnere data, og så:

```suggestion
    queryFn: () => getRecruitmentApplicationsForApplicant(recruitmentId as string),
```

Reply:
https://github.com/Samfundet/Samfundet4/pull/1595/commits/129d3a9de0b3f72dc376c2c29a65b64c6497ade9